updatecli/updatecli.d/jdk11.yaml

```diff
@@ -1,78 +0,0 @@
----
-name: Bump JDK11 version
```

Comment:
No need for updatecli manifest anymore? I'm thinking about these images: https://github.com/jenkins-infra/docker-inbound-agents/pull/47/files#diff-ef76acb1714bcdd754ddc63df9bfd8bc84bf763aef7e4bdbab1b95cef4b3c5e6R5

Reply:
erf...

> to come: updatecli manifest for each jdk